baselines/webworker.generated.d.ts

```diff
@@ -2691,19 +2716,23 @@ declare var ReadableByteStreamController: {
 interface ReadableStream<R = any> {
     readonly locked: boolean;
     cancel(reason?: any): Promise<void>;
+    getReader(options: { mode: "byob" }): ReadableStreamBYOBReader;
     getReader(): ReadableStreamDefaultReader<R>;
+    getReader(options?: ReadableStreamGetReaderOptions): ReadableStreamReader<R>;
     pipeThrough<T>(transform: ReadableWritablePair<T, R>, options?: StreamPipeOptions): ReadableStream<T>;
     pipeTo(destination: WritableStream<R>, options?: StreamPipeOptions): Promise<void>;
     tee(): [ReadableStream<R>, ReadableStream<R>];
 }
 
 declare var ReadableStream: {
     prototype: ReadableStream;
+    new(underlyingSource: UnderlyingByteSource, strategy?: { highWaterMark?: number }): ReadableStream<Uint8Array>;
+    new<R = any>(underlyingSource: UnderlyingDefaultSource<R>, strategy?: QueuingStrategy<R>): ReadableStream<R>;
     new<R = any>(underlyingSource?: UnderlyingSource<R>, strategy?: QueuingStrategy<R>): ReadableStream<R>;
```

Comment:
These additional constructor signatures make these examples work:
```javascript
new ReadableStream({
  start(controller) {
    controller.enqueue("a");
  }
});

new ReadableStream({
  type: "bytes",
  start(controller) {
    controller.byobRequest?.respond(1);
  }
});
```
(And yes, there are unit tests. 😉)

How do we feel about this? I think it's worth the extra complexity, since these are literally the textbook examples on how developers can construct a readable stream.